builtin/logical/pki/dnstest/server.go

```diff
@@ -64,9 +64,10 @@ func (ts *TestServer) setupRunner(domain string, network string) {
 		ContainerName: "bind9-dns-" + strings.ReplaceAll(domain, ".", "-"),
 		NetworkName:   network,
 		Ports:         []string{"53/udp"},
-		LogConsumer: func(s string) {
-			ts.log.Info(s)
-		},
+		// DNS container logging was disabled to reduce content within CI logs.
```

Comment:
Should this be removed?

Reply:
Yes we will try it for now, it was another request to reduce logging in CI from Paul Banks.